frontend/src/views/Lobby.tsx

```diff
@@ -142,6 +142,7 @@ function LobbyPage() {
   const [active, setActive] = useState(false);
   const [difficulty, setDifficulty] = useState<Difficulty | null>(null);
   const [duration, setDuration] = useState<number | undefined>(15);
+  const [size, setSize] = useState<number | undefined>(4);
```

Comment:
nit: Thoughts on making the default larger, maybe even like `10`? This would require a change to the backend as well, though I'm open on this.

Reply:
+1 on this, I remember when we played a game with the TeamsCode team we unexpectedly ran into this room limit haha (and from that demo it seems the service works fine with 7-8 people as long as we don't hit the max questions error, so I think my previous concern about that no longer exists)